approved/v93k/prb1.tf

```diff
@@ -1887,35 +1859,30 @@ test_flow
     if @G200_864CE8F_FAILED == 1 then
     {
       {
+        @G100_864CE8F_RAN = 1;
```

Comment:
New if_ran implementation for a group, set the flag immediately upon entering the group, rather than pushing it down to individual tests.